cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -877,6 +879,24 @@ func (k *KubernetesClient) isResourceNamespaced(gvk schema.GroupVersionKind) (bo
 	return false, errors.New(fmt.Sprintf("group version kind not found: %v", gvk.String()))
 }
 
+func (k *KubernetesClient) mapRole(group, version, kind, name, namespace string ) (*dto.Role, error){
+	 role, err := k.clientset.RbacV1().Roles(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	 
+	 if err != nil {
+		return nil, err
```

Comment:
Make sure to format the code with `go fmt ./...`

Reply:
Doesn't the commit command formats the code, I have seen it do something but I am not sure if it formatting process

Reply:
We have it in place only for the UI